error_middleware.go

```diff
@@ -29,7 +30,7 @@ var ErrorMiddleware MiddlewareFunc = MiddlewareFunc(func(handler HandlerFunc) Ha
 				}
 				log.WithError(err).Error("recover panic")
 				w.WriteHeader(500)
-				fmt.Fprintln(w, err)
+				_, _ = fmt.Fprintln(w, err)
```

Comment:
I don't like these

Reply:
Should I go back over the others with `json.NewEncoder...` ?

Reply:
Yes please

Reply:
Done